mindmaps-migration/base/src/main/java/io/mindmaps/migration/base/AbstractMigrator.java

```diff
@@ -34,7 +34,7 @@
 public abstract class AbstractMigrator implements Migrator {
 
     public static final int BATCH_SIZE = 25;
-    public final QueryBuilder queryBuilder = Graql.withoutGraph();
+    public final QueryBuilderImpl queryBuilder = (QueryBuilderImpl) Graql.withoutGraph();
```

Comment:
Another PR introduced this `registerMacro(Macro macro)` to `QueryBuilderImpl` and I did not add that to the interface. Hence the casting.